### ISSUE TITLE
[Bug Report] Possible error with thumbnail download.

### ISSUE THREAD

**Issue by Syoker** [JunkFood02/Seal#244]
### Describe the bug

When I run a thumbnail download (either with a custom command or with the download thumbnail button), the file does indeed download, but no messaging service detects the file, be it WhatsApp, Telegram or Discord. But if I rename the file (with anything, adding an "a" at the end or deleting any character) the file appears in the gallery. Could this be a bug in the app?

### To Reproduce

1.Download a thumbnail (with the button or with a custom command)
2. Access any messaging app and enter the image gallery of the same app (not the file explorer).
3. And that's it, the image is not going to be there.

### Device info

App version: 1.3.0 (10300)
Device information: Android 12 (API 31)
[arm64-v8a, armeabi-v7a, armeabi]
Screen resolution: 1517 x 719

**Comment by JunkFood02** [comment]
Not very related with this app. Android will scan the files regularly into their media library which can be accessed by the other apps. The renaming operation just triggered a scanning.

**Comment by JunkFood02** [comment]
Install [this build](https://github.com/JunkFood02/Seal/actions/runs/3095850230), thumbnails downloaded in general mode should be scanned to the media library now

**Comment by Syoker** [comment]
Thank you for your hard work.

I tried the build you did, and now with the button to save thumbnails the thumbnail does appear in the messenger apps. But if I run a custom command, there is still the same problem.

https://user-images.githubusercontent.com/61765318/191551968-ea75c0fb-58cd-44cc-a746-e0355c62ff1b.mp4

The command I used was as follows: `--write-thumbnail --convert-thumbnails png --skip-download
`

Thank you for trying to fix this problem!
